### PR TITLE
Ros2

### PR DESCRIPTION
**Pull request by Kapim**

### CODE REVIEW

File: era_5g_relay_network_application/server.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from typing import Any, Dict, Optional, Tuple, Union
 
 import rclpy  # pants: no-infer-dep
+from rclpy.parameter import Parameter
```
Comment (ZdenekM) [inline era_5g_relay_network_application/server.py]: Add `# pants: no-infer-dep`?

File: mypy.ini [hunk 2]
```diff
@@ -75,3 +75,6 @@ ignore_missing_imports = True
 
 [mypy-example_interfaces.*]
 ignore_missing_imports = True
+
+[mypy-ros2bag.*]
```
Comment (ZdenekM) [inline mypy.ini:79]: New dependency - should be added here https://github.com/5G-ERA/relay_network_application/blob/ros2/.github/workflows/pants.yaml#L29? Maybe not, if it it in the base installation of ROS.

**Review by ZdenekM — CHANGES_REQUESTED** [review]

File: mypy.ini (hunk 2 above)
Comment (Kapim) [reply mypy.ini:79]: according to the official documentation it is part of any ros-base installation 

> rosbag2 packages are available via debian packages, and will already be included with any -ros-base installation

File: era_5g_relay_network_application/server.py (hunk 1 above)
Comment (Kapim) [reply era_5g_relay_network_application/server.py]: added

**Review by ZdenekM — APPROVED** [review]

**Pull request merged**